src/core/modes/walk-mode.ts

```diff
@@ -60,18 +60,37 @@ export class WalkMode implements Mode {
       return;
     }
 
-    const position = firstVisibleRange.start;
-    const range = new vscode.Range(position, position);
+    let visibleStartLine = firstVisibleRange.start;
+    let visibleEndLine = firstVisibleRange.start;
+    const junbaeLocation = { top: '5%', 'font-size': '50px', bottom: 'none' };
+    const timerLocation = { top: '5%', bottom: 'none' };
+    const location = vscode.workspace.getConfiguration('junbae-mode').get('location');
+
+    if (location === 'bottom') {
+      visibleStartLine = firstVisibleRange.end;
+      visibleEndLine = firstVisibleRange.end;
+      junbaeLocation.top = 'none';
+      junbaeLocation.bottom = '10px';
+      junbaeLocation['font-size'] = '100px';
+      timerLocation.top = 'none';
+      timerLocation.bottom = '70px';
+    }
+
+    const range = new vscode.Range(visibleStartLine, visibleEndLine);
 
     if (this.combo !== this.renderedComboCount || !range.isEqual(this.renderedRange!)) {
       this.renderedComboCount = this.combo;
       this.renderedRange = range;
-      this.createWalkMotionDecorator([range], editor);
-      this.createTimerDecoration([range], editor);
+      this.createWalkMotionDecorator([range], editor, junbaeLocation);
+      this.createTimerDecoration([range], editor, timerLocation);
     }
   }
 
-  private createWalkMotionDecorator = (ranges: vscode.Range[], editor: vscode.TextEditor) => {
+  private createWalkMotionDecorator = (
+    ranges: vscode.Range[],
+    editor: vscode.TextEditor,
+    junbaeLocation: { top: string; ['font-size']: string; bottom: string },
```

Comment:
why should it receive by parameters?

Reply:
Because there are parts where attributes are determined by location config values (junbae and timer location)